packages/ui/src/proposals/constants/proposals.ts

```diff
@@ -0,0 +1,5 @@
+import { ProposalStage } from '../types'
+
+export const proposalStages: ProposalStage[] = ['DECIDING', 'DORMANT', 'GRACING', 'SUCCEEDED']
```

Comment:
This looks like duplicating the `ProposalStage` type from \`[packages/ui/src/proposals/types/proposals.ts](https://github.com/Joystream/pioneer/pull/673/files#diff-a8741635086cb8ca5e34592af0f1627dbe7f55e8828ffe1bac1cd4113182cca8) \`.